test/integration/user_can_view_past_orders_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
+
+class UserCanViewPastTrips < ActionDispatch::IntegrationTest
+  test 'user can view past trips' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.